src/components/createRecipe/recipeForm/Step4.jsx

```diff
@@ -17,16 +12,28 @@ import {
   AddItem
 } from "./FormStyled.styles";
 
+import React, { useState } from "react";
+import { connect } from "react-redux";
+import * as dispatchers from "../../../actions/actionCreators"
+
+
 function Step4(props) {
   const { goForward, goBackward, addInstructionsToBody } = props;
+
   const [inputState, setInputState] = useState("");
+
   const [instructionsArray, setInstructionsArray] = useState([]);
 
   const inputHandler = e => {
     e.preventDefault();
     setInputState(e.target.value);
   };
 
+
+  const goBack = e => {
```

Comment:
I've seen this function a few times now. Perhaps it should be passed down in props, not redefined within each component?

Reply:
good idea, but I will probably do it later when the more urgent functionalities will be working!